EIPS/eip-7872.md

```diff
@@ -0,0 +1,44 @@
+---
+eip: 7872
+title: Max blob flag for local builders
+description: Adds a flag to set the maximum number of blobs a local builder will put in a block
+author: Francesco D'Amato <francesco.damato@ethereum.org>, Kevaundray Wedderburn (@kevaundray), Toni Wahrstätter (@nerolation), Alex Stokes (@ralexstokes), Ben Adams (@benaadams), Gajinder Singh (@g11tech), Dustin (@tersec)
+discussions-to: https://ethereum-magicians.org/t/max-blob-flags-for-local-builders/22734
+status: Draft
+type: Meta
+created: 2025-01-30
+---
+
+## Abstract
+
+This EIP adds a flag to the block builder in order to allow them to include a client configured maximum amount of blobs.  
+
+## Motivation
+
+Currently a builder will include all blobs in their local mempool, up to the maximum amount that the protocol requires. If a builder has low bandwidth, they may include too many blobs
```

Comment:
Proposer bandwidth is a problem we must solve with PeerDAS - and I think we will solve it with distributed blob building (which is mostly _distributed blob publishing_ after #9378) , because local block builders will be pretty useless with propagating large number of blobs anyway - anything more than **half of the electra max blobs** (due to erasure coding) will require a higher bandwidth for them today - which is **only 4.5 blobs under PeerDAS** without distributed blob building.